.github/workflows/create-release.yml

```diff
@@ -1,96 +1,78 @@
-name: "Create release"
+name: Create release
 
 on:
   workflow_dispatch:
+    inputs:
+      DRY_RUN:
+        type: boolean
+        default: false
+        description: "Dry run: setting this to `true` will skip the final step of publishing the release, leaving it as a draft-release. This can be helpful for testing and debugging. Note that the tag and release are still getting generated and the next run of this result in incremented release version. To avoid this, do not forget to delete the resulting release and tag."
 
 jobs:
-  verify-release:
-    name: Verify release
-    runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.gen-version.outputs.VERSION }}
+  gen-version:
+    name: Generate semantic version from branch and tags
+    uses: kyma-project/eventing-tools/.github/workflows/get-version-from-release-branch-reusable.yml@main
+    # outputs: VERSION
 
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
+  bump-sec-scanners-config:
+    name: Bump the sec-scandners-config.yaml
+    needs: gen-version
+    uses: kyma-project/eventing-tools/.github/workflows/bump-sec-scanners-config-reusable.yml@main
```

Comment:
(nit) ignore for now, after noticing how the reusable workflows are used, maybe it would have been better to use the following naming instead of the current one:

```
kyma-project/eventing-tools/.github/workflows/reusable/bump-sec-scanners-config.yaml
```

but this is only my personal preference, so feel free to ignore.

Reply:
That might have been a helpful solution. You would still find workflows at the expected place and the extra dir would make clear what this is, especially when we are calling them from other workflow like in this case. Learnings for the future.